project-extender/src/providers/jira-cloud-provider/JiraCloudProvider.ts

```diff
@@ -216,12 +216,9 @@ class JiraCloudProvider implements ProviderApi {
     return response
   }
 
-  async getIssuesBySprintAndProject(
-    sprintId: number,
-    project: string
-  ): Promise<Issue[]> {
+  async getIssuesBySprintAndProject(sprintId: number): Promise<Issue[]> {
```

Comment:
we have to refactor this to "getIssuesBySprint", since the project parameter is not used anymore